### PR TITLE
remove show-checksums flag for dvc status commmand

### LINKED ISSUE #2085: status: remove  `--show-checksums`
**Please provide information about your setup**
DVC version(i.e. `dvc --version`), Platform and method of installation (pip, homebrew, pkg Mac, exe (Windows), DEB(Linux), RPM(Linux))

Currently, `status --show-checksums` will have same output as `status`. Make it display as described in [options](https://dvc.org/doc/commands-reference/status#options).

### PR DESCRIPTION
**Pull request by kurianbenoy**
Fix #2085 
* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x ] Does your PR affect documented changes or does it add new functionality
      that should be documented?

* Fix changes by updating documentation.https://github.com/iterative/dvc.org/pull/514
-----

### CODE REVIEW

**Comment by shcheklein** [comment]
@kurianbenoy as we discussed, it should be removed across other command as well that support this option. Also, please, check docs and examples in the corresponding files.

File: tests/func/test_data_cloud.py [hunk 1]
```diff
@@ -507,18 +507,18 @@ def _test_cloud(self, remote=None):
         cache_dir = stage_dir.outs[0].cache_path
 
         # FIXME check status output
-        self.main(["status", "-c", "--show-checksums"] + args)
+        #        self.main(["status", "-c", "--show-checksums"] + args)
```
Comment (mroutis) [inline tests/func/test_data_cloud.py]: If this isn't needed it is better to delete it than commenting it (the same for the comments below)

File: dvc/repo/status.py [hunk 2]
```diff
@@ -74,10 +73,8 @@ def _cloud_status(
     )["local"]
 
     ret = {}
-    status_info = self.cloud.status(
-        used, jobs, remote=remote, show_checksums=show_checksums
-    )
-    for md5, info in status_info.items():
+    status_info = self.cloud.status(used, jobs, remote=remote)
+    for _, info in status_info.items():
```
Comment (efiop) [inline dvc/repo/status.py]: I guess we could use `status_info.values()` instead, but at that point we might refactor this so it is efficiently using filter() and map(), but that would be a separate task. Having this as-is is fine for now.

**Comment by efiop** [comment]
@kurianbenoy Also, please either submit a PR to fix docs or at least create an issue for it on dvc.org. Thank you! :slightly_smiling_face:

**Comment by efiop** [comment]
For the record, travis is failing due to unrelated homebrew issue. All tests are passing fine.

As to codeclimate, it is complaining about issues that are not related to this patch and has been there before. So no action required.

File: dvc/repo/status.py (hunk 2 above)
Comment (kurianbenoy) [reply dvc/repo/status.py]: I have made the changes. @efiop I am curious what is the use of files in  directory `command` and `status` in this project.

Also I couldn't find any command with data_sync.py in folder `command`. Why is it like that?

File: dvc/repo/status.py [hunk 3]
```diff
@@ -74,10 +73,8 @@ def _cloud_status(
     )["local"]
 
     ret = {}
-    status_info = self.cloud.status(
-        used, jobs, remote=remote, show_checksums=show_checksums
-    )
-    for md5, info in status_info.items():
+    status_info = self.cloud.status(used, jobs, remote=remote)
+    for _, info in status_info.values():
```
Comment (efiop) [inline dvc/repo/status.py]: you probably meant
```suggestion
    for info in status_info.values():
```

File: dvc/repo/status.py (hunk 2 above)
Comment (efiop) [reply dvc/repo/status.py]: @kurianbenoy

> I have made the changes.

Please see another comment down below.

>  I am curious what is the use of files in directory command and status in this project.
> Also I couldn't find any command with data_sync.py in folder command. Why is it like that?

just legacy files. push/pull/fetch/status were called "data sync" a while ago.

Comment (kurianbenoy) [reply dvc/repo/status.py]: Ok thanks for clarrifying

**Pull request merged**